src/routes/teamsRoutes.js

```diff
@@ -1,79 +1,164 @@
 const express = require("express");
-const { isLoggedIn, isInstructor } = require("../middlewares/middleware");
+const { isLoggedIn, isInstructor, isStudent } = require("../middlewares/middleware");
```

Comment:
The `isStudent` middleware is imported but never used. If it's unnecessary, remove it to clean up unused dependencies and avoid linting errors. If it will be used in future routes, include a comment explaining its purpose.